src/firebaseDb.ts

```diff
@@ -13,7 +13,7 @@ export let dbInstance: Firestore
 const testFirestoreIsInitialized = async (newDbInstance: Firestore) => {
     try {
         const ref = newDbInstance.collection('facilities')
-        //eslint-disable-next-line @typescript-eslint/no-unused-vars
+        //eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
         const dbResult = await ref.limit(1).get()
```

Comment:
Why not remove  the `dbResult` variable since it's not being used?

Reply:
It is being used to verify we can access the database. I guess we can remove the left side of the equation though 🤔

Reply:
Yeah, I don't think we need to declare the variable here if it's not being used anywhere else. Could just do the `await ref` without it.